tests/ecosystem/deployment/test_deployment.py

```diff
@@ -162,3 +163,13 @@ def ocs_install_verification():
         item['metadata']['name'] for item in storage_classes['items']
     }
     assert required_storage_classes.issubset(storage_class_names)
+
+    # Verify OSD's are distributed
+    log.info("Verifying OSD's are distributed evenly across worker nodes")
+    ocp_pod_obj = ocp.OCP(kind=constants.POD, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    osds = ocp_pod_obj.get(selector='app=rook-ceph-osd')['items']
+    node_names = [osd['spec']['nodeName'] for osd in osds]
+    for node in node_names:
```

Comment:
we need to check with worker node that is labeled for ocs, I guess this might fail if we have more then 3 worker

Reply:
If we have more than 3 workers, and extra worker is part OCS cluster, only OSD will be created in 4th node. here we are making sure each osd is created on unique node for deployments. So this logic will apply for all AZ's as well as for all platforms.

Reply:
Is this really resolved? @prsurve ?

Reply:
yes